pentaho-i18n-webservice-bundle/pom.xml

```diff
@@ -28,6 +28,16 @@
       <groupId>javax.ws.rs</groupId>
       <artifactId>javax.ws.rs-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>jakarta.xml.ws</groupId>
+      <artifactId>jakarta.xml.ws-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.sun.xml.ws</groupId>
+      <artifactId>jaxws-ri</artifactId>
+      <version>2.3.3</version>
```

Comment:
Can this version get pushed to the maven parent pom?

Reply:
So far this is the only dependency on this library throughout all the repositories. If I find it is used elsewhere we can pull it up to the parent poms.